### PR TITLE
match format dates to browser prefer language

### PR DESCRIPTION
**Pull request by wiolaSzczepanik**

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "history": "^4.6.3",
     "marked": "^0.3.6",
-    "moment": "^2.24.0",
+    "moment": "^2.8.1",
```
Comment (Mequrel) [inline webapp/package.json]: Please revert the change. The moment was already in higher version

File: webapp/src/components/DateRange.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+    const userLang = navigator.language || navigator.userLanguage;
+    const language = userLang.split("-", 1);
+
     const start = moment(new Date(props.start));
     const end = moment(new Date(props.end));
 
+    start.locale(language);
+    end.locale(language);
+
+    console.log("Language: " + language);
```
Comment (Mequrel) [inline webapp/src/components/DateRange.js]: please remove the debug info

File: webapp/src/components/DateRange.js [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
+    moment.locale(language);
```
Comment (Mequrel) [inline webapp/src/components/DateRange.js]: this is not needed when you call locale on start and end separately

**Review by Mequrel — CHANGES_REQUESTED** [review]

File: webapp/package.json (hunk 1 above)
Comment (wiolaSzczepanik) [reply webapp/package.json]: Done :)

File: webapp/src/components/DateRange.js (hunk 2 above)
Comment (wiolaSzczepanik) [reply webapp/src/components/DateRange.js]: Done :)

File: webapp/src/components/DateRange.js (hunk 3 above)
Comment (wiolaSzczepanik) [reply webapp/src/components/DateRange.js]: Done:)
